pwndbg/commands/__init__.py

```diff
@@ -264,6 +268,18 @@ def fix_reraise(*a, **kw) -> str | pwndbg.dbg_mod.Value | None:
     return fix(*a, reraise=True, **kw)  # type: ignore[misc]
 
 
+def fix_reraise_arg(arg) -> pwndbg.dbg_mod.Value:
+    """fix_reraise wrapper for evaluating command arguments"""
+    try:
+        # Will always return pwndbg.gdb_mod.Value because
```

Comment:
Shouldn't we say pwndbg.dbg_mod.Value? I guess in LLDB this won't return gdb_mod :P
```suggestion
        # Will always return pwndbg.dbg_mod.Value because
```

Reply:
Probably a typo. `gdb_mod` doesn't exist lol.

Reply:
Oh oops, yup it's a typo